config/config.example.yml

```diff
@@ -1,27 +1,7 @@
 ## Combined configuration definition to setup objects at Cowbird startup
-## See documentation 'Configuration'
+## See documentation 'Configuration' for more details on each section.
 
 # [Required] Definition of managed services by Cowbird.
-#
-# services:
-# ---------
-#   [Required] Definition of managed services by Cowbird. Each service is provided as string that must match an actual
-#   implementation in Cowbird. Each service must be further configured with one or more of the following parameters. If
-#   a required parameter is missing for a service it will throw a `ServiceConfigurationException` exception.
-#   For more details on available services on Magpie, refer to these pages :
-#     - https://pavics-magpie.readthedocs.io/en/latest/services.html#available-services
-#     - https://pavics-magpie.readthedocs.io/en/latest/autoapi/magpie/services/index.html
-#
-#   service:
-#     active:             [optional, default=False] bool allowing to deactivate a service and stop managing it
-#     priority:           [optional, default=math.inf] Relative priority between services while handling events
-#                         Lower values have higher priority, default value is last
-#     url:                [optional, default=None] URI of the web service represented by this Cowbird service.
-#                         Some Cowbird services do not represent web service, but other will throw an exception if
-#                         missing.
-#     workspace_dir:      [optional, default=None] Location of the users workspace root.
-#                         Required for the following services : FileSystem, Catalog and Geoserver
-#
```

Comment:
I think this definition should remain in the file as well for quick reference. 
The official doc can provide the fields in a better format (table?) and add more details if needed.